src/Menu/Menu.utils.ts

```diff
@@ -73,11 +73,11 @@ export const menuDefaultPositionSetter: MenuPositionSetter = ({
     case 'vertical-stretched': {
       let minWidth: number | undefined
 
-      const bottomOfTrigger = triggerDimensions.bottom + MARGIN
+      const bottomOfTrigger = triggerDimensions.bottom
```

Comment:
again this PR should not about styling 🤔

Reply:
This is related to the changes I needed to operate in order for options to stay opened while hovering.